src/v2/Components/NavBar/Menus/UserMenu.tsx

```diff
@@ -39,39 +42,59 @@ export const UserMenu: React.FC = () => {
   const hasPartnerAccess = Boolean(user.has_partner_access)
 
   return (
-    <Menu onClick={trackClick}>
-      {isAdmin && <MenuItem href={sd.ADMIN_URL}>Admin</MenuItem>}
-      {(isAdmin || hasPartnerAccess) && (
-        <MenuItem href={sd.CMS_URL}>CMS</MenuItem>
+    <Menu m={0}>
```

Comment:
If all of these now require a `m={0}` should we fix it internal to the component?

Reply:
Ah yeah, I had assumed this was being used elsewhere. The design is really inconsistent with the full width dropdowns as well.

Reply:
Yah a few different people worked on this stuff so i would expect some inconsistency. Feel free to update; the these menus in force aren't being used anywhere else.